### PR TITLE
readme: lint and add instruction for destroy

### PR DESCRIPTION
**Pull request by vnghia**
Adding instruction for destruction because we have some issues related to it

### CODE REVIEW

File: terraform/ecs/agent/README.md [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
+  - `export AWS_ACCESS_KEY_ID=`
+  - `export AWS_SECRET_ACCESS_KEY=`
+  - `export AWS_REGION=$ECS_CLUSTER_REGION`
+- `terraform init -backend-config="bucket=$S3_BUCKET_BACKEND" -backend-config="key=$S3_KEY_BACKEND"`
+  - We are setting S3 backend to maintain the terraform state. you could use existing s3 bucket (make sure it's private)
+    - ex: `terraform-state-prod`
+  - provide unique key for storing terraform state.
+    - ex: `ecs/packetai-agent/$CLUSTER_NAME.tfstate`
+- Run: `terraform apply -var="cluster_name=$CLUSTER_NAME" -var="X_PAI_TOKEN=$X_PAI_TOKEN" -var="PAI_API_KEY=$PAI_API_KEY" -var="APIDEVURL=$APIDEVURL"`
+- To destroy, because of [#4852](https://github.com/hashicorp/terraform-provider-aws/issues/4852), we must run `aws autoscaling update-auto-scaling-group --auto-scaling-group-name "$(terraform output -raw asg_name)" --min-size 0 --desired-capacity 0` before running `terraform apply -var="cluster_name=$CLUSTER_NAME" -var="X_PAI_TOKEN=$X_PAI_TOKEN" -var="PAI_API_KEY=$PAI_API_KEY" -var="APIDEVURL=$APIDEVURL"`
```
Comment (gara-MI) [inline terraform/ecs/agent/README.md]: @vnvo2409 We won't destroy the cluster, we will only remove the packetai-agent, so why do we need to scale down the nodes ?.

File: terraform/ecs/agent/README.md [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
+  - `export AWS_ACCESS_KEY_ID=`
+  - `export AWS_SECRET_ACCESS_KEY=`
+  - `export AWS_REGION=$ECS_CLUSTER_REGION`
+- `terraform init -backend-config="bucket=$S3_BUCKET_BACKEND" -backend-config="key=$S3_KEY_BACKEND"`
+  - We are setting S3 backend to maintain the terraform state. you could use existing s3 bucket (make sure it's private)
+    - ex: `terraform-state-prod`
+  - provide unique key for storing terraform state.
+    - ex: `ecs/packetai-agent/$CLUSTER_NAME.tfstate`
+- Run: `terraform apply -var="cluster_name=$CLUSTER_NAME" -var="X_PAI_TOKEN=$X_PAI_TOKEN" -var="PAI_API_KEY=$PAI_API_KEY" -var="APIDEVURL=$APIDEVURL"`
+- To destroy, `terraform apply -var="cluster_name=$CLUSTER_NAME" -var="X_PAI_TOKEN=$X_PAI_TOKEN" -var="PAI_API_KEY=$PAI_API_KEY" -var="APIDEVURL=$APIDEVURL"`
```
Comment (gara-MI) [inline terraform/ecs/agent/README.md]: @vnvo2409 you should change it to destroy: `terraform apply`

File: terraform/ecs/lambda/README.md [hunk 3]
```diff
@@ -1,13 +1,15 @@
...
+  - `export AWS_ACCESS_KEY_ID=`
+  - `export AWS_SECRET_ACCESS_KEY=`
+  - `export AWS_REGION=$ECS_CLUSTER_REGION`
+- `terraform init -backend-config="bucket=$S3_BUCKET_BACKEND" -backend-config="key=$S3_KEY_BACKEND"`
+  - We are setting S3 backend to maintain the terraform state. you could use existing s3 bucket (make sure it's private)
+    - ex: `terraform-state-prod`
+  - provide unique key for storing terraform state.
+    - ex: `ecs/packetai-lambda/$CLUSTER_NAME.tfstate`
+- Run: `terraform apply -var="cluster_name=$CLUSTER_NAME" -var="INFRA_ID=$INFRA_ID" -var="INGEST_URL=$INGEST_URL"`
+- To destroy, run `terraform apply -var="cluster_name=$CLUSTER_NAME" -var="INFRA_ID=$INFRA_ID" -var="INGEST_URL=$INGEST_URL" -destroy`
```
Comment (gara-MI) [inline terraform/ecs/lambda/README.md:15]: @vnvo2409 same here, should change to destroy right ?

Comment (vnghia) [reply terraform/ecs/lambda/README.md:15]: well, actually, `terraform destroy` is `terraform apply -destroy`

Comment (gara-MI) [reply terraform/ecs/lambda/README.md:15]: never mind I see destroy at the end.

**Review by gara-MI — APPROVED** [review]

**Pull request merged**